README.md

```diff
@@ -128,13 +129,17 @@ then Lodash/Underscore is the better option.*
 **[Lang](#lang)**
 
 1. [_.isNaN](#_isnan)
+1. [_.isUndefined](#_isundefined)
+1. [_.gt](#_gt)
+1. [_.gte](#_gte)
```

Comment:
Is those (`isUndefined / gt / gte / isArrayBuffer`) related `get` ?

I think better that split to another pull request if those not related.

Reply:
@zeeshanu Could you separate them into different PR like what @terrierscript  suggested? thanks. :smile: